### PR TITLE
Space DnD for HomingPenguin

### PR DESCRIPTION
**Pull request by Kokojos**
Think of this as a PR made by #Homingpenuins

This can be closed : https://github.com/TheSwain/Fulpstation/pull/225
 ## About The Pull Request
 This adds a spawner ment to play dnd. you spawn in with all chameleon gear to roleplay.
 
 ## Why It's Good For The Game
 It is yet another spawner for people to play with when they inevidably die.
 
 ## Changelog
 🆑
 add: Added new things: Mansion.dmm
 code: changed some code: Fulpstationruins.dm
 /🆑

### CODE REVIEW

**Comment by Kokojos** [comment]
Urgh, I can't figure out why this is causing a conflict? Fulpstationruins should already by changed.

**Comment by Kokojos** [comment]
This has been tested on my test server, @TheSwain

**Comment by Homingpenguins** [comment]
thanks for doing whatever tag you did to make the spawner menu actually correct. can you show me sometime what you did to do that?

File: code/FulpstationCode/fulpspaceruins.dm [hunk 1]
```diff
@@ -116,10 +117,14 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
 	mob_name = "Syndicate engineer"
 	icon = 'icons/obj/machines/sleeper.dmi'
 	icon_state = "sleeper_s"
-	flavour_text = "<span class='big bold'>Welcome, syndicate engineer,</span><b> You have been charged by your superiors to repair a derelict station and ensure it can support human life. You're pretty sure nobody will disturb you here, yet you can't shake the feeling this job will be like no other <b>Your superiors are watching you closely, <font size=4>DO NOT</font> leave the station lest you disappoint your employer!</b>"
+
+	short_desc = "Syndicate engineer, You have been charged by your superiors to repair a derelict station"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: period?

File: code/FulpstationCode/fulpspaceruins.dm [hunk 2]
```diff
@@ -116,10 +117,14 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
...
+	flavour_text = "You must ensure that the station can support human life. You're pretty sure nobody will disturb you here, yet you can't shake the feeling this job will be like no other."
+	important_info = "Leaving the station is forbidden"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: period?

File: code/FulpstationCode/fulpspaceruins.dm [hunk 3]
```diff
@@ -142,6 +147,61 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
...
+	return ..()
+
+/obj/effect/mob_spawn/human/Dndplayer
+    name = "DnD player"
+    roundstart = FALSE
+    death = FALSE
+    random = TRUE
+    icon = 'icons/obj/machines/sleeper.dmi'
+    icon_state = "sleeper_s"
+    short_desc = "You are a very avid DnD player"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: period?

File: code/FulpstationCode/fulpspaceruins.dm [hunk 4]
```diff
@@ -142,6 +147,61 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
...
+    flavour_text = "You volunteered to go around in a specialised ship, playing DnD until you die."
+    important_info = "It is forbidden to attack other players and to not bathe"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: might wanna clean up the wording of this a bit

File: code/FulpstationCode/fulpspaceruins.dm [hunk 5]
```diff
@@ -142,6 +147,61 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
...
+	suit = /obj/item/clothing/suit/chameleon
+	shoes = /obj/item/clothing/shoes/chameleon
+	gloves = /obj/item/clothing/gloves/chameleon
+	ears = /obj/item/radio/headset/chameleon
+	back = /obj/item/storage/backpack/chameleon
+	belt = /obj/item/storage/belt/chameleon
+	id = /obj/item/card/id/syndicate/anyone
+
+/obj/effect/mob_spawn/human/DnDMaster
+    name = "DnDMaster"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: space

File: code/FulpstationCode/fulpspaceruins.dm [hunk 6]
```diff
@@ -142,6 +147,61 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
 /datum/outfit/syndicate_engineer/post_equip(mob/living/carbon/human/H)
 	H.faction |= ROLE_SYNDICATE
 
+/obj/effect/mob_spawn/human/syndicate_engineer/Destroy()
+	new/obj/structure/fluff/empty_sleeper/syndicate(get_turf(src))
+	return ..()
+
+/obj/effect/mob_spawn/human/Dndplayer
+    name = "DnD player"
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm:155]: capitalization

File: code/FulpstationCode/fulpspaceruins.dm (hunk 4 above)
Comment (ATH1909) [reply code/FulpstationCode/fulpspaceruins.dm]: also, period?

File: code/FulpstationCode/fulpspaceruins.dm [hunk 7]
```diff
@@ -167,4 +227,11 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
...
-	description = "Apparently, the first station built by NanoStrasen, but the Syndicates may want to reclaim it."
\ No newline at end of file
+	description = "Apparently, the first station built by NanoStrasen, but the Syndicates may want to reclaim it."
+
+/datum/map_template/ruin/space/dndmansion
+	id = "dndmansion"
+	suffix = "DnDmansion.dmm"
+	name = "dndmansion"
+	placement_weight = 3 //Fun spawners in space are rare. Might be toned down in the future.
+	description = "a rich guy hired a bunch of nerds so he can watch you play dnd.That nerd is you. have fun."
```
Comment (ATH1909) [inline code/FulpstationCode/fulpspaceruins.dm]: Capitalization.

Also, add a space after that first   P E R I O D , please.

Comment (Homingpenguins) [reply code/FulpstationCode/fulpspaceruins.dm]: that punctuation mastake was actually my fault.

File: code/FulpstationCode/fulpspaceruins.dm (hunk 5 above)
Comment (Homingpenguins) [reply code/FulpstationCode/fulpspaceruins.dm]: i need to learn coding etiquette

**Comment by Kokojos** [comment]
Thanks @ATH1909 for the comments, it's been fixed.

**Comment by remove32** [comment]
requested a fix for an oddly placed tile on the map. once that's in we should be good to go on this one

**Comment by Homingpenguins** [comment]
alright. i put the fixed dmm in the dev_features chat. pinged you and kokojo. and ask you to fix it. also. is that stale label bad?

**Comment by Kokojos** [comment]
Done

**Comment by Homingpenguins** [comment]
thanks for the fix

**Comment by Fikou** [comment]
you should run this through map merger

**Comment by remove32** [comment]
> 
> 
> you should run this through map merger

wish we had any CI stuff set up

this is a new ruin though, nothing to merge against

**Pull request merged**